### PR TITLE
AEA-3034 Provide a clear schema for how dosage instructions should be populated

### PR DESCRIPTION
**Pull request by kris-szlapa**

### CODE REVIEW

**Review by TomSmithCoding — DISMISSED** [review]
Good work Kris!

**Review by MatthewPopat-NHS — CHANGES_REQUESTED** [review]
This should be reviewed from a content perspective before merging

File: packages/specification/schemas/components/MedicationDispense/MedicationDispense.yaml [hunk 1]
```diff
@@ -212,10 +212,41 @@ properties:
     items:
       type: object
       properties:
+        sequence:
+          type: integer
+          format: int32
+          example: 1
         text:
           type: string
           description: The dosage instructions expressed as text.
```
Comment (AmeyaKrishnamoorthy) [inline packages/specification/schemas/components/MedicationDispense/MedicationDispense.yaml:221]: Need to clarify that multiple dose text strings will be concatenated into the final prescription.

**Review by AmeyaKrishnamoorthy — CHANGES_REQUESTED** [review]

File: packages/specification/schemas/components/MedicationDispense/MedicationDispense.yaml (hunk 1 above)
Comment (kris-szlapa) [reply packages/specification/schemas/components/MedicationDispense/MedicationDispense.yaml]: Added to text description

File: packages/specification/schemas/components/MedicationRequest/MedicationRequest.yaml [hunk 2]
```diff
@@ -202,10 +202,41 @@ properties:
...
+          type: integer
+          format: int32
+          example: 1
         text:
           type: string
           description: The dosage instructions expressed as text.
-          example: 4 times a day - Oral
+          example: 4 times a day - for 4 days - Oral - Dissolved under the tongue.
+        additionalInstruction:
+          type: object
```
[the remark this answers is not in the document]
Comment (kris-szlapa) [reply packages/specification/schemas/components/MedicationRequest/MedicationRequest.yaml]: The type changed and items added

**Review by AmeyaKrishnamoorthy — APPROVED** [review]

File: packages/specification/schemas/components/MedicationDispense/MedicationDispense.yaml [hunk 3]
```diff
@@ -212,10 +212,41 @@ properties:
...
-          example: 4 times a day - Oral
+          example: 4 times a day - for 4 days - Oral - Dissolved under the tongue.
+        additionalInstruction:
+          type: object
```
[the remark this answers is not in the document]
Comment (kris-szlapa) [reply packages/specification/schemas/components/MedicationDispense/MedicationDispense.yaml]: Type changed to array

**Pull request merged**